bool found_solution

float32 rvecs1
float32 rvecs2
float32 rvecs3

float32 tvecs1
float32 tvecs2
float32 tvecs3

float32 bottom_left_x
float32 bottom_left_y
float32 bottom_middle_left_x
float32 bottom_middle_left_y
float32 bottom_middle_right_x
float32 bottom_middle_right_y
float32 bottom_right_x
float32 bottom_right_y

float32 top_left_x
float32 top_left_y
float32 top_middle_left_x
float32 top_middle_left_y
float32 top_middle_right_x
float32 top_middle_right_y
float32 top_right_x
float32 top_right_y
